src/commands/Suggestions/resolvesuggestion.ts

```diff
@@ -0,0 +1,122 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed, TextChannel } from 'discord.js';
+import { SkyraCommand } from '@lib/structures/SkyraCommand';
+import { PermissionLevels } from '@lib/types/Enums';
+import { GuildSettings } from '@lib/types/settings/GuildSettings';
+import { isNumber } from '@klasa/utils';
+import { SuggestionData } from '@lib/types/definitions/Suggestion';
+
+const enum SuggestionsColors {
+	ACCEPTED = '4CB02C',
+	CONSIDERED = 'CFA08D',
+	DENIED = 'F90505'
+}
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['resu', 'rs'],
+			cooldown: 10,
+			description: language => language.tget('COMMAND_RESOLVESUGGESTION_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_RESOLVESUGGESTION_EXTENDED'),
+			flagSupport: true,
+			permissionLevel: PermissionLevels.Moderator,
+			requiredPermissions: ['EMBED_LINKS'],
+			runIn: ['text'],
+			usage: '<suggestion:suggestion> <accept|a|deny|d|consider|c> [comment:...string]',
+			usageDelim: ' '
+		});
+
+		this.createCustomResolver('suggestion', async (arg, _, message) => {
+			const channelID = message.guild!.settings!.get(GuildSettings.Suggestions.SuggestionsChannel);
+			if (channelID === null) throw message.language.tget('COMMAND_SUGGEST_NOSETUP', message.author.username);
+			const channel = this.client.channels.get(channelID) as TextChannel;
+
+			const id = parseInt(arg, 10);
```

Comment:
For faster (and much safer, see `Number('12a')` vs `parseInt('12a', 10)`) parsing:
```suggestion
			const id = Number(arg);
```